presto-plugin-toolkit/src/test/java/com/facebook/presto/plugin/base/security/TestAllowAllAccessControl.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.base.security;
+
+import com.facebook.presto.spi.connector.ConnectorAccessControl;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Method;
+
+import static java.lang.String.format;
+
+public class TestAllowAllAccessControl
+{
+    @Test
+    public void testEverythingImplemented()
+            throws ReflectiveOperationException
+    {
+        checkEverythingImplemented(ConnectorAccessControl.class, AllowAllAccessControl.class);
+    }
+
+    static <I> void checkEverythingImplemented(Class<I> interfaceClass, Class<? extends I> implementationClass)
```

Comment:
Is it different than `InterfaceTestUtils`'s?

Reply:
sorry for the late reply!  I didn't see your comment.  presto-plugin-toolkit can't access InterfaceTestUtils.

Reply:
but they're the same

Reply:
Please add a comment in both implementations that they need to be the same.